### PR TITLE
Create en_US translation properties

### PR DESCRIPTION
**Pull request by jb2barrels**
Copied from en_GB. Fixed variables noFavourites and home.pdfOrganiser.title to use its US corrected spelling.

### CODE REVIEW

**Comment by Frooodle** [comment]
Can you also add flag svg and navbar.html change?

File: src/main/resources/messages_en_US.properties [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+
+
+#merge
+merge.title=Merge
+merge.header=Merge multiple PDFs (2+)
+merge.submit=Merge
+
+
+#pdfOrganiser
+pdfOrganiser.title=Page Organiser
```
Comment (Frooodle) [inline src/main/resources/messages_en_US.properties]: z need here

File: src/main/resources/messages_en_US.properties [hunk 2]
```diff
@@ -0,0 +1,728 @@
...
+    
+                                   
+#pdfToImage
+pdfToImage.title=PDF to Image
+pdfToImage.header=PDF to Image
+pdfToImage.selectText=Image Format
+pdfToImage.singleOrMultiple=Image result type
+pdfToImage.single=Single Big Image
+pdfToImage.multi=Multiple Images
+pdfToImage.colorType=Colour type
```
Comment (Frooodle) [inline src/main/resources/messages_en_US.properties]: color

File: src/main/resources/messages_en_US.properties [hunk 3]
```diff
@@ -0,0 +1,728 @@
...
+pdfToImage.color=Colour
+pdfToImage.grey=Greyscale
```
Comment (Frooodle) [inline src/main/resources/messages_en_US.properties]: gray

**Review by Frooodle — CHANGES_REQUESTED** [review]

**Comment by jb2barrels** [comment]
@Frooodle Thank you for the additional US language correction point outs.
As requested I've also added the US svg and navbar changes.

**Review by Frooodle — APPROVED** [review]

**Comment by Frooodle** [comment]
Hi jb2barrels,

We are updating Stirling-PDF's license to MPL 2.0 (Mozilla Public License Version 2.0) to continue fostering our open-source commitment while introducing more flexible usage of the project.

### What We Need From You:
In order to transition to the MPL 2.0 license, we need the explicit consent of all our contributors. 
1. If you're comfortable relicensing your contributions for this specific PR under MPL 2.0, please respond with:
`
I, jb2barrels, grant permission to relicense my contributions from this PR to Stirling-PDF under the Mozilla Public License 2.0. 
`
2. If you'd prefer to give blanket permission for all of your contributions to Stirling-PDF (be it through pull requests, commits, or any other form of contribution) rather than on a PR-by-PR basis, please respond with:
`
I, jb2barrels, grant permission to relicense all my contributions to Stirling-PDF under the Mozilla Public License 2.0.
`

### What does this do and why?
As your old code was contributed under a different license it legally cannot move to a new license without your permission, and since we dont want to lose your code we wanted to reach out. Remember this project is remaining open-source!.

### Deadline:
If we dont receive a response, or if you decline, unfortunately, we will have to remove your contributions from Stirling-PDF before the license change, which is scheduled  in coming months.

### Thank You! 
Your contributions are invaluable, and we hope to continue having them in Stirling-PDF under the new license. Feel free to ask if you have any questions or concerns about this change.

Best,
Frooodle
Stirling-PDF Maintainer

**Comment by jb2barrels** [comment]
I, jb2barrels, grant permission to relicense my contributions from this PR to Stirling-PDF under the Mozilla Public License 2.

**Pull request merged**